test/WebJobs.Script.Tests.Integration/EndToEndTestFixture.cs

```diff
@@ -143,7 +144,13 @@ protected virtual void CreateTestStorageEntities()
 
         public virtual void Dispose()
         {
-            Host.Stop();
+            try
+            {
+                Host.Stop();
+            }
+            catch
+            {
+            }
```

Comment:
The SB listener throws because it's receiver is null due to the incomplete start? Yeah, I think we should pull those changes into this PR before getting it in. I'll go sign off on the other change. Incorrectly handling Stop (allowing it to end prematurely before allowing all listeners a chance to stop) would be bad.